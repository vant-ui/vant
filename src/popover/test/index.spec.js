// import Popover from '..';
// import { mount } from '../../../test';
