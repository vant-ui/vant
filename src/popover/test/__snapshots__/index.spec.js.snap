// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should locate to reference element when showed 1`] = `<!---->`;

exports[`should locate to reference element when showed 2`] = `
<div class="van-popup van-popover van-popover--light" name="van-popover-zoom" style="position: absolute; left: 0px; top: 8px; margin: 0px; z-index: 2005;" data-popper-placement="bottom">
  <div class="van-popover__arrow"></div>
  <div class="van-popover__content"></div>
</div>
`;

exports[`should locate to reference element when showed 3`] = `
<div class="van-popup van-popover van-popover--light" name="van-popover-zoom" style="position: absolute; left: 0px; top: 8px; margin: 0px; z-index: 2005; display: none;" data-popper-placement="bottom">
  <div class="van-popover__arrow"></div>
  <div class="van-popover__content"></div>
</div>
`;

exports[`should watch placement prop and update location 1`] = `
<div class="van-popup van-popover van-popover--light" name="van-popover-zoom" style="z-index: 2006; position: absolute; left: 0px; top: -8px; margin: 0px;" data-popper-placement="top">
  <div class="van-popover__arrow"></div>
  <div class="van-popover__content"></div>
</div>
`;
